controller/config/beerocks_controller.conf.in

```diff
@@ -38,6 +38,9 @@ load_monitor_on_vaps=1
 #   health check feature:
 load_health_check=0
 
+#   statistics polling feature
+load_statistics_polling=1
```

Comment:
We already have a configuration to enable/disable the statistics polling task: `load_diagnostics_measurements`.
So this commit is redundant.

Reply:
ok